sdk/storage/azure-storage-file-share/azure/storage/fileshare/_parser.py

```diff
@@ -39,6 +39,7 @@ def _parse_datetime_from_str(string_datetime):
 
 
 def _datetime_to_str(datetime_obj):
+    from azure.storage.fileshare._generated._serialization import Serializer
```

Comment:
Any reason to import this here rather than at the top? Here means we're going to import it every time we call this function.

Reply:
I had in my note "local import", so I thought this is what we meant 😄

Reply:
Ah right, yeah I think that actually referred doing a relative import like `from ._generated._serialization import Serializer` which is something you'll need to do too.